### PR TITLE
LPS-199861 Second level child Site doesn't show the name of the parent

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-199861)

## Steps 

1. Go to Control Panel > Sites > Create a site 
2. Create a child site inside the previous site
3. See the button titles in the child site (parent site name) and in the parent site (Go to Sites label)

Screenshot 2023-10-24 at 15 25 46
Screenshot 2023-10-24 at 15 25 59

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -22,7 +22,15 @@ if (group != null) {
...
-	portletDisplay.setURLBackTitle(portletDisplay.getPortletDisplayName());
+
+	if (Validator.isNull(group.getParentGroup())) {
+		portletDisplay.setURLBackTitle(portletDisplay.getPortletDisplayName());
+	}
+	else {
+		Group parentGroup = group.getParentGroup();
+
+		portletDisplay.setURLBackTitle(parentGroup.getDescriptiveName(locale));
+	}
```
Comment (BarbaraCabrera) [inline modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/view.jsp:33]: Hey! I think we can simplify this. Maybe we can let the setURLBackTitle as it was and doing something like: 
```
Group parentGroup = group.getParentGroup();

if (Validator.isNotNull(parentGroup)) {
	portletDisplay.setURLBackTitle(parentGroup.getDescriptiveName(locale));
}
```

**Comment by DiegoHu97** [comment]
@BarbaraCabrera Updated! 😄

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/142791

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**